.github/actions/issue-labeler/src/index.mjs

```diff
@@ -0,0 +1,80 @@
+// @ts-check
+import * as github from '@actions/github'
+import * as core from '@actions/core'
+import { readFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+const __dirname =
+  '/home/runner/work/next.js/next.js/.github/actions/issue-labeler'
```

Comment:
IMHO we should use `$GITHUB_ACTION_PATH` and `path.join` here.

See https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
Yeah especially because the double next.js in the path seems like a bug